Song_sort_Genre.php

```diff
@@ -1,39 +1,70 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title>Layout1Musicplay</title>
+		<title>Sort By Genre</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
+		<!--Connected to css -->
 		<link rel ="stylesheet" href = "test12.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
-		<div class="img_1">
-			<img src = "images/img.png">
-		</div>
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
-			<nav>
-				<ul>
-					<li><a href = "index.html">Home</a></li>
-					<li><a href = "index.html">Music Play</a></li>
-					<li><a href = "index.html">Contact</a></li>
-					<li><a href = "01_login_real.php">Account</a></li>
-				</ul>
-			</nav>
-		</div>
-		<div class="content_1"><!-- Holds the main page content -->
-			<div class="section3"><!-- Holds the main page connect -->
-				<h1>All the song upload in this playlist.Total duration of playlist:</h1>
+			<!-- Set a div that contain logo for the music database -->
```

Comment:
21/9 because I has delete the nav bar above so I need to called up a nav bar (normal nav because this page for user.) So I need to called up the normal div box and also add a logo in the nav bar